DZSwiftUI/DZSwiftUI/Halpers/ShakeEffect.swift

```diff
@@ -0,0 +1,30 @@
+// ShakeEffect.swift
+// Copyright © RoadMap. All rights reserved.
+
+import SwiftUI
+
+/// Анимация неверного ввода
+struct ShakeEffect: GeometryEffect {
+    // MARK: - Private Constants
+
+    private enum Constants {
+        static let travelDistance: CGFloat = 6
+        static let numOfShake: CGFloat = 4
+        static let translationY: CGFloat = 0
+    }
+
+    // MARK: - Public Properties
+
+    var travelDistance = Constants.travelDistance
+    var numOfShake = Constants.numOfShake
```

Comment:
```suggestion
    var travelDistance = Constants.travelDistance
    var numOfShake = Constants.numOfShake
```
```suggestion
    var travelDistance = Constants.travelDistance
    var numOfShake = Constants.numOfShake
```
приватными пробовал сделать?

Reply:
пробовал, ругается